README.md

```diff
@@ -342,6 +342,35 @@ Marqo is a community project with the goal of making tensor search accessible to
 
 5. If you update dependencies, make sure to delete the .tox dir and rerun.
 
+## Redis set up
```

Comment:
Developing locally doesn't actually require Redis, right? I've been able to run Marqo locally without redis running

Let's keep this README lightweight, and point possible contributors to the dev guide if they want a more detailed dev getting started.

Reply:
It's not, but I thought people might get confused as to why they get redis errors when they try to run Marqo locally without it.